content/millennium/r4/diagnostic/observation.md

```diff
@@ -0,0 +1,122 @@
+
+### Parameters
+
+ Name             | Required?         | Type          | Description
+------------------|-------------------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------
+ `patient`        | This or `subject` | [`reference`] | The subject that the observation is about (if patient). Example: `patient=1316024`
+ `subject`        | This or `patient` | [`reference`] | The subject (Patient) that the observation is about. Example: `subject=Patient/1316024` or `subject:Patient=1316024`
+ `code`           | N                 | [`token`]     | The code or component-code of the observation type. Example: `code=http://loinc.org|3094-0,http://loinc.org|3139-3`
+ `date`           | N                 | [`date`]      | Date range into which the observation falls. Example: `date=gt2014-09-24` or `date=lt2015-09-24T12:00:00.000Z`
+ `_lastUpdated`   | N                 | [`date`]      | Date range in which the observation was last updated. Example: `_lastUpdated=gt2014-09-24` or `_lastUpdated=lt2015-09-24T12:00:00.000Z`
+ `category`       | N                 | [`token`]     | The category of observations. Example: `category=laboratory`
+ [`_count`]       | N                 | [`number`]    | The maximum number of results to return per page. Defaults to `50`.
+
+
+
+Notes:
+
+* The `subject` parameter must represent a Patient resource and may use the `:Patient` modifier.
+
+* It is recommended to search by either `code` or `date` (or both).
+
+* The `code` parameter:
+  * May be a list of comma separated values. A system must be provided for each code.
+  * Searches only `Observation.code`. For example when fetching blood pressures the resource will be only be returned when the search is based on `85354-9(Systolic and Diastolic BP)`. Using the component codes `8480-6(Systolic BP)` or `8462-4 (Diastolic BP)` will not return the resource.
+
+* The `date` and `_lastUpdated` parameters may be provided up to two times, and must use the `eq`, `ge`, `gt`, `le`, or `lt` prefixes. When a value is provided without a prefix, an implied `eq` prefix is used. When provided twice, the lower value must have a `ge` or `gt` prefix and the higher value must have an `le` or `lt` prefix.
+
+* The `date` and `_lastUpdated` parameters may not be provided together.
+
+* The `_lastUpdated` query will only qualify clinically significant updates. For example, changes to the value or code, and other significant fields. Minor updates, like some non-clinically relevant note updates, will not qualify.
+
+### Headers
+
+ <%= headers %>
+
+### Example
+
+#### Request
+
+    GET https://fhir-open.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Observation?patient=3998008
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:R4_OBSERVATION_BUNDLE) %>
+
```

Comment:
let's add the disclaimer here.
https://github.com/cerner/fhir.cerner.com/blame/master/content/millennium/r4/financial/coverage.md#L79

Reply:
https://github.com/cerner/fhir.cerner.com/pull/345/commits/3756e931418b52a36c0e6998015daa0d7adbb1c5#diff-d42a2575832583fdd3d20dff4a54f294R110